spec/components/discourse_spec.rb

```diff
@@ -142,7 +142,7 @@ def name
             'pluginauth'
           end
 
-          def enabled
```

Comment:
I think this can be removed because it is not called anyway (the test would have failed if it had been).